translations/management/commands/add_translations.py

```
@@ -186,17 +202,60 @@ def handle(self, *args, **options):
             # "__all__" expands to every configured non-default language inside the client.
-            results = translate.bulk_translate(["__all__"], unique_texts)
+            # strict=False so one language mangling a placeholder does not cost us the
+            # other sixteen; the failures are reported below and simply get no row,
+            # which falls back to English.
+            results = translate.bulk_translate(["__all__"], unique_texts, strict=False)
+        except TranslationIntegrityError as e:
+            raise CommandError(
+                f"A translation came back with its placeholders altered; English rows were saved "
+                f"but no other-language rows were written. This is the guard working - the string "
+                f"needs a human translation rather than a re-run. Error: {e}"
+            )
```

Ruff (0.16.2)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)